pdb2pqr/pdb2pqr/main.py

```diff
@@ -431,9 +435,12 @@ def non_trivial(args, protein, definition, is_cif):
             _LOGGER.info("Assigning titration states with PROPKA.")
             protein.remove_hydrogens()
             pka_df, pka_filename = run_propka(args, protein)
-            _LOGGER.error("\n" + str(pka_df))
-            raise NotImplementedError(
-                "PROPKA not implemented. See %s" % pka_filename)
+
+            protein.apply_pka_values(forcefield_.name, args.ph, dict(zip(pka_df.group_label, pka_df.pKa)));
+
+            # _LOGGER.error("\n" + str(pka_df))
```

Comment:
Can you remove these commented-out lines, please?

Reply:
What happened to following the PEP8 style guide?  I just looked at this code and it has both a semicolon and an overly long line.

Reply:
I fixed this with https://github.com/Electrostatics/apbs-pdb2pqr/pull/605/commits/0ee632b7c49e86276053dce64a8c3fde42c5d8c7